plotly/tests/test_optional/test_jupyter/package.json

```diff
@@ -15,6 +15,8 @@
     "ecstatic": "^2.1.0",
     "tap-parser": "^2.0.0",
     "tape": "^4.6.0",
-    "xhr": "^2.2.2"
+    "xhr": "^2.2.2",
+    "electron": "^1.8.4",
+    "orca": "^1.2.1"
```

Comment:
I wouldn't have expected this to be needed with the scraper tests living under test_orca. Was this a holdover from earlier experiments?